test/integration/02-user-wallet/02-receive-lightning.spec.ts

```diff
@@ -56,21 +151,32 @@ describe("UserWallet - Lightning", () => {
 
     const hash = getHash(invoice)
 
-    await pay({ lnd: lndOutside1, request: invoice })
-
-    expect(
-      await Wallets.updatePendingInvoiceByPaymentHash({
+    const updateInvoice = () =>
+      Wallets.updatePendingInvoiceByPaymentHash({
         paymentHash: hash as PaymentHash,
         logger: baseLogger,
-      }),
-    ).not.toBeInstanceOf(Error)
+      })
+
+    const promises = Promise.all([
```

Comment:
I'm assuming we're doing this instead of separate `await`s to get the tests to run faster?

Reply:
just to use the same pattern as above. this is not stricly needed here. it's needed in the test above because this is the only way to get the result from `pay`